vendor/bat-native-ledger/src/bat/ledger/internal/wallet/wallet_util.cc

```diff
@@ -0,0 +1,189 @@
+/* Copyright (c) 2021 The Brave Authors. All rights reserved.
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+#include "bat/ledger/internal/wallet/wallet_util.h"
+
+#include "base/json/json_reader.h"
+#include "base/json/json_writer.h"
+#include "base/strings/string_number_conversions.h"
+#include "base/strings/stringprintf.h"
+#include "bat/ledger/global_constants.h"
+#include "bat/ledger/internal/ledger_impl.h"
+#include "bat/ledger/internal/state/state_keys.h"
+
+namespace ledger {
+namespace wallet {
+
+namespace {
+
+std::string WalletTypeToState(const std::string& wallet_type) {
+  if (wallet_type == constant::kWalletBitflyer) {
```

Comment:
another reason for the enum is that we can have a switch statement here

Reply:
also prefer `else if` even with a return, it's just easier to read that way imo

Reply:
fixed